spring-thymeleaf-2/src/main/java/com/baeldung/thymeleaf/mvcdata/repository/EmailData.java

```diff
@@ -0,0 +1,50 @@
+package com.baeldung.thymeleaf.mvcdata.repository;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmailData implements Serializable {
+    private static final String EMAIL_SUBJECT = "You have received a new message";
```

Comment:
So this class looks more familiar to our readers, let's please follow the Java bean convention, like so:

```java
public class EmailData implements Serializable {
    private String subject;
    private String content;

    // ...

    public String getSubject() {
        return this.subject;
    }

    // ...
}
```

instead of using a `repository` list and constants.